docs/sphinx/user_guide/feature/atomic.rst

```diff
@@ -31,12 +31,12 @@ A complete working example code that shows RAJA atomic usage can be found in
 Atomic Operations
 -----------------
 
-RAJA atomic support includes a range of the most common atomic operations.
+RAJA atomic support includes a variety of the most common atomic operations.
 
 .. note:: * Each RAJA atomic operation is templated on an *atomic policy*.
-          * Each of methods below returns the value of the potentially modified
-            argument (i.e., \*acc) immediately before the atomic operation is 
-            applied.
+          * Each of methods described in the table below returns the value of 
```

Comment:
"Each of _the_ methods" or "Each method".

Reply:
Changed to shorter version.